app/src/main/java/com/brainwallet/ui/theme/Color.kt

```diff
@@ -0,0 +1,226 @@
+package com.brainwallet.ui.theme
+
```

Comment:
will change it later with the brainwallet color palettes

Reply:
do you agree we have  to heavily edit the color theme?  The xml ia a complete mess.  I have gotten blocked debugging it to just have it Light and Dark (or day /night)

Reply:
do you mean the color theme from the `colors.xml` ? if yes, IMHO no need to heavily edit, but will check on it next week, I will help to dentify and update the parent style instead, maybe will just using default color theme/scheme from material